### PR TITLE
pull request for changes in osganimation (including MorphGeometry)

### PR DESCRIPTION
**Pull request by mp3butcher**
Lot of stuff
This pull request is mainly for merging of osganimation changes:
-Cedric Pinson repo integration
-Coverity warning fixes

### CODE REVIEW

File: src/osgAnimation/RigGeometry.cpp [hunk 1]
```diff
@@ -24,15 +24,15 @@ osg::BoundingBox RigComputeBoundingBoxCallback::computeBound(const osg::Drawable
 {
     const osgAnimation::RigGeometry& rig = dynamic_cast<const osgAnimation::RigGeometry&>(drawable);
 
-    // if a valid initial bounding box is set we use it without asking more
+    // if a valid inital bounding box is set we use it without asking more
```
Comment (filnet) [inline src/osgAnimation/RigGeometry.cpp]: !

File: src/osgAnimation/RigGeometry.cpp [hunk 2]
```diff
@@ -24,15 +24,15 @@ osg::BoundingBox RigComputeBoundingBoxCallback::computeBound(const osg::Drawable
...
     if (rig.getInitialBound().valid())
         return rig.getInitialBound();
 
     if (_computed)
         return _boundingBox;
 
     // if the computing of bb is invalid (like no geometry inside)
-    // then don't tag the bounding box as computed
+    // then dont tag the bounding box as computed
```
Comment (filnet) [inline src/osgAnimation/RigGeometry.cpp]: !

File: src/osgAnimation/RigGeometry.cpp [hunk 3]
```diff
@@ -70,8 +69,8 @@ RigGeometry::RigGeometry(const RigGeometry& b, const osg::CopyOp& copyop) :
     _vertexInfluenceMap(b._vertexInfluenceMap),
     _needToComputeMatrix(b._needToComputeMatrix)
 {
-    // we don't copy the RigImplementation yet. because the RigImplementation need to be initialized in a valid graph, with a skeleton ...
-    // don't know yet what to do with a clone of a RigGeometry
+    // we dont copy the RigImplementation yet. because the RigImplementation need to be initialized in a valid graph, with a skeleton ...
+    // dont know yet what to do with a clone of a RigGeometry
```
Comment (filnet) [inline src/osgAnimation/RigGeometry.cpp]: !

**Comment by openscenegraph** [comment]
I have done a manual merge from Julien's zip file submission to osg-users

**Pull request closed without merging**